pkg/api/controller.go

```diff
@@ -339,7 +339,7 @@ func (c *Controller) StartBackgroundTasks(reloadCtx context.Context) {
 	if c.Config.Storage.SubPaths != nil {
 		for route, storageConfig := range c.Config.Storage.SubPaths {
 			// Enable running garbage-collect periodically for subImageStore
-			if storageConfig.GC && storageConfig.GCInterval != 0 {
+			if storageConfig.GC {
```

Comment:
This is changing behavior right?
storageConfig.GCInterval != 0 means we can turn off GC with this set to zero? We don't want this anymore?

Reply:
I though we decided in one of our previous discussions periodic GC would always be enabled if inline GC is removed.

Reply:
I tested this case now and it looks like gc will be run continuously

Reply:
You still want the user to be able to disabled GC?

Reply:
> I tested this case now and it looks like gc will be run continuously

Sorry, tested again and I was wrong first time(I think I used another config). If GCInterval=0 set explicitly, then because of this check from scheduler https://github.com/project-zot/zot/blob/main/pkg/scheduler/scheduler.go#L344 gc is executed only one time

Reply:
In this case I think we can leave the implementation as is in this PR:
- enable/disabled GC periodically by using the 'enable' setting
- if unspecified, the GC is enabled with 1h interval by default
- if GC is enabled and 0 is specified as interval, run GC only once
- if GC is enabled an a non-0 interval value is used, run GC periodically at the specified interval

Reply:
This is a good doc item
cc: @mbshields